examples/simple-chatbot/client/javascript/src/app.js

```diff
@@ -129,6 +179,49 @@ class ChatbotClient {
 
     // Set up listeners for media track events
     this.setupTrackListeners();
+
+    await this.pcClient.initDevices();
+    this.setupEventListeners();
+    this.registerFunctionCallHandlers();
+  }
+
+  registerFunctionCallHandlers() {
+    this.pcClient.registerFunctionCallHandler(
```

Comment:
here's the interesting v2 bit, demonstrating how to register function call handlers on the client.